runtime/lua/editorconfig.lua

```diff
@@ -215,11 +271,11 @@ local function parse(filepath, dir)
   return opts
 end
 
---- Configure the given buffer with options from an .editorconfig file
----
----@param bufnr integer Buffer number to configure
----
----@private
+local M = {}
+
+--- @private
```

Comment:
Should we keep the convention of prefixing private methods with an underscore?

Reply:
Maybe. Arguably this function isn't actually private though so I left it as is for now.

I'm not sure if these needs either `@nodoc` or `@package`, or something else entirely.